src/common/constants.js

```diff
@@ -0,0 +1,32 @@
+const LOG_MESSAGE = Object.freeze({
+  START_MESSAGE: "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분",
+  ATTEMPT_COUNT_MESSAGE: "시도할 횟수는 몇 회인가요?",
+  WINNERS_ANNOUNCEMENT: '최종 우승자 : ',
+})
+
+const ERROR_MESSAGE = Object.freeze({
+  INVALID_INPUT: "자동차 이름은 최대 5글자까지 가능합니다. 다시 입력해 주세요.",
+  EMPTY_INPUT: "입력값이 비어 있습니다. 값을 입력해 주세요!",
+  DUPLICATE_INPUT: "중복된 자동차 이름이 있습니다. 서로 다른 이름으로 입력해 주세요!",
+  INVALID_ATTEMPTS_COUNT: "시도 횟수는 1 이상의 정수여야 합니다. 다시 입력해 주세요!",
+});
+
+const TAG = Object.freeze({
+  ERROR: "[ERROR]",
+});
+
+const GAME_RULES = Object.freeze({
+  MOVE_THRESHOLD: 4,
+  DISTANCE_SYMBOL: '-',
+  DELIMITER: ',',
+  MAX_CAR_NAME_LENGTH: 5,
+  MIN_IN_RANDOM_NUMBER: 0,
+  MAX_IN_RANDOM_NUMBER: 9,
+});
+
+export {
+  ERROR_MESSAGE, 
+  LOG_MESSAGE,
+  TAG,
+  GAME_RULES,
```

Comment:
```suggestion
  ERROR_MESSAGES, 
  LOG_MESSAGES,
  TAGS,
  GAME_RULES,
```
객체로 선언되어 있어서 `GAME_RULES`처럼 복수형태라는 걸 네이밍으로 알려줘도 좋을 것 같아요!

`TAG`도 지금 당장은 하나의 프로퍼티만 가지고 있지만 객체로 선언하신 만큼 충분히 늘어날 수 있는 여지가 있다고 생각해서 이것도 복수형태의 네이밍을 작성해도 괜찮을 것 같네요🙃